Source/CarthageKit/GitHub.swift

```diff
@@ -64,55 +64,142 @@ extension GitHubError: Decodable {
 	}
 }
 
-/// Describes a GitHub.com repository.
+/// Describes a GitHub.com or GitHub Enterprise repository.
 public struct GitHubRepository: Equatable {
+
+	/// Represents a URL for a GitHub website.
+	public enum URL: Equatable, Hashable, Printable {
```

Comment:
Actually, `Server` is probably better yet. That's the name that octokit.objc uses.